app/components/post-status.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const {
+  computed,
+} = Ember;
+
+/**
+  A badge that presents a post's status
+
+  ## default usage
+
+  ```handlebars
+  {{post-status post=post}}
+  ```
+
+  @class post-status
+  @module Component
+  @extends Ember.Component
+ */
+export default Ember.Component.extend({
+  classNames: ['post-meta'],
```

Comment:
I would rename this to match the `post-status`, which will likely mean a CSS change. Not sure if there's a `post-meta` scss file.